CHANGELOG.md

```diff
@@ -21,7 +21,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   commits. The list won't show more than 10 commits to not clutter the console.
 
 * `jj unsquash` has been removed in favor of `jj squash` and
-  `jj diffedit --restore-descendants`.
+  `jj diffedit --preserve-descendant-contents`.
```

Comment:
TODO: I'm planning to change this to `--preserve-descendant-content` (without the s at the end), unless we go with something completely different. This matches the help message better, and I don't think the `s` helps much.